hackathon/erh-guardian-agent/ui/src/styles.css

```
@@ -0,0 +1,69 @@
+}
+.card h2 { margin: 0 0 12px; font-size: 15px; text-transform: uppercase; letter-spacing: 0.04em; color: var(--muted); }
+.stat { display: flex; align-items: baseline; gap: 8px; margin-bottom: 8px; }
+.big { font-size: 30px; font-weight: 700; }
+.muted { color: var(--muted); }
+ul { margin: 4px 0 0; padding-left: 18px; }
+.tablewrap { overflow-x: auto; }
+table { width: 100%; border-collapse: collapse; }
+th, td { text-align: left; padding: 8px 10px; border-bottom: 1px solid var(--border); vertical-align: top; }
+th { color: var(--muted); font-weight: 600; font-size: 13px; }
+.mono { font-family: ui-monospace, monospace; font-size: 13px; white-space: nowrap; }
+.action { max-width: 340px; word-break: break-word; font-size: 13px; }
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)